spec/factories/form_submission_attempts.rb

```diff
@@ -29,9 +29,8 @@
       aasm_state { 'pending' }
     end
 
-    trait :form_upload do
-      association :form_submission,
-                  form_data: { email: 'a@b.com', full_name: { first: 'Veteran', last: 'Eteranvay' } }.to_json
+    trait :nil_form_data do
+      association :form_submission, form_data: nil
```

Comment:
The `form_upload` trait was more or less meaningless so we're replacing it with a `nil` form data one.